src/components/InputField/InputFieldStyled.js

```diff
@@ -17,7 +18,16 @@ export const InputFieldStyled = styled.div`
 `;
 
 export const StyledLabel = styled.label`
-  color: ${getNotificationColor};
+  position: absolute;
+  left: 1px;
+  transform: translateY(-50%);
+  background: var(--white-color);
+  padding: 0px 4px 0px 4px;
+  margin: 0 12px;
+  color: #49454F;
```

Comment:
Same lets add it to the GlobalStyles

Reply:
Changed all text colors to var(--color-grey-6)